LLama/Native/NativeApi.Load.cs

```diff
@@ -155,13 +155,13 @@ private static List<string> GetLibraryTryOrder(NativeLibraryConfig.Description c
             if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
                 platform = OSPlatform.Windows;
-                prefix = "runtimes/win-x64/native/";
```

Comment:
I don't think we should change this path. We used to put files under the `runtimes/{system}/native` folder because they could be detected without specifying the dll path. Though the feature detection actually specify the library path manually, I prefer to just keep them unless any bad impact is confirmed. 🤣

Reply:
Done 👍